aspnet/fundamentals/localization.rst

```diff
@@ -1,8 +1,285 @@
-.. include:: /../common/stub-topic.txt
+ASP.NET Core Internationalization: Localization and Globalization 
+==================================================================
 
-|stub-icon| Localization
-========================
+By `Rick Anderson`_ with assistance from `Damien Bowden <https://twitter.com/damien_bod>`_, `Bart Calixto <https://twitter.com/bartmax>`_, and `Nadeem Afana <https://twitter.com/NadeemAfana>`__ 
 
-.. include:: /../common/stub-notice.txt
+Creating a multilingual website with ASP.NET Core will allow your site to reach a wider audience. ASP.NET has :doc:`/fundamentals/request-features` that provide support for localizing into different languages and cultures.
```

Comment:
I'm not sure why the Request Features doc is relevant here. How about: "ASP.NET Core provides services and middleware for localizing into different languages and cultures;"